README.md

```diff
@@ -20,6 +20,7 @@ Available addons
 addon | version | summary
```

Comment:
There is one README for the repo but you should have one for the module 
as well.